util/object.go

```diff
@@ -0,0 +1,91 @@
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This should have the standard Contour header.

Reply:
@stevesloka this is a known [issue](https://github.com/projectcontour/contour-operator/issues/11). I'll fix the header across the entire project in a separate PR that closes the issue.